inc/tag_replacer.php

```diff
@@ -210,11 +217,16 @@ public function process_image_tags( $content, $images = [] ) {
 			);
 
 			// If the image is in header or has a class excluded from lazyload, we need to do the regular replace.
-			if ( $images['in_header'][ $index ] ) {
+			if ( $images['in_header'][ $index ] || self::$lazyload_skipped_images < $this->settings->get( 'skip_lazyload_images' ) ) {
```

Comment:
The index check is not properly placed, it should be inside `can_lazyload_for` as last condition since `regular_tag_replace` from within `lazyload_tag_replace`